codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/generators/EnumGenerator.java

```diff
@@ -15,26 +15,24 @@
 
 package software.amazon.smithy.ruby.codegen.generators;
 
-import java.util.List;
-import java.util.stream.Collectors;
+import java.util.Map;
 import software.amazon.smithy.codegen.core.directed.GenerateEnumDirective;
-import software.amazon.smithy.model.shapes.Shape;
-import software.amazon.smithy.model.traits.EnumDefinition;
-import software.amazon.smithy.model.traits.EnumTrait;
+import software.amazon.smithy.model.shapes.EnumShape;
 import software.amazon.smithy.ruby.codegen.GenerationContext;
 import software.amazon.smithy.ruby.codegen.RubyFormatter;
 import software.amazon.smithy.ruby.codegen.RubySettings;
+import software.amazon.smithy.ruby.codegen.generators.docs.ShapeDocumentationGenerator;
 import software.amazon.smithy.utils.SmithyInternalApi;
 import software.amazon.smithy.utils.StringUtils;
 
 @SmithyInternalApi
 public final class EnumGenerator extends RubyGeneratorBase {
 
-    private final Shape shape;
+    private final EnumShape shape;
 
     public EnumGenerator(GenerateEnumDirective<GenerationContext, RubySettings> directive) {
         super(directive);
-        this.shape = directive.shape();
+        this.shape = (EnumShape) directive.shape();
```

Comment:
Use expectEnumShape instead here

Reply:
I had that originally but I copied what int enum shape did. I can change both